app/controllers/papers_controller.rb

```diff
@@ -40,11 +42,19 @@ def update
       paper.update(update_paper_params)
     end
 
+    notify_paper_edited! if params[:paper][:locked_by_id].present?
+
     respond_with paper
   end
 
   # non RESTful routes
 
+  def activity_feed
+    # TODO: params[:name] probably needs some securitifications
+    activity_feeds = ActivityFeed.where(feed_name: params[:name], subject_id: paper.id).order('created_at DESC')
```

Comment:
Line is too long. [112/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.